ambry-rest/src/main/java/com.github.ambry.rest/RestServer.java

```diff
@@ -233,7 +235,7 @@ public void start() throws InstantiationException {
       restResponseHandler.start();
```

Comment:
Had an offline discussion with Ze. `restResponseHandler` and `restRequestHandler` are actually same instance within `restHandlerFactory`. Hence, calling start() once should suffice. Let's leave a comment here to explain we removed `restRequestHandler.start();`. In the future, we probably could unify them.